extensions/MESA/EGL_MESA_swap_control_tear.txt

```diff
@@ -0,0 +1,164 @@
+Name
+
+    MESA_swap_control_tear
+
+Name Strings
+
+    EGL_MESA_swap_control_tear
+
+Contributors
+
+    Adam Jackson <ajax@redhat.com>
+    Daniel Kartch
+    Ethan Lee
+    Shahbaz Youssefi
+
+    The contributors to GLX_EXT_swap_control_tear:
+    Brian Nguyen, NVIDIA, brnguyen at nvidia.com
+
+Contact
+
+    Adam Jackson <ajax@redhat.com>
+
+Status
+
+    DRAFT
+
+Version
+
+    Version 2 (29 September 2020)
+
+Number
+
+    XXX
+
+Extension type
+
+    EGL display extension
+
+Dependencies
+
+    EGL 1.1 or greater is required for eglSwapInterval.
+
+    This specification is written against the EGL 1.5 specification.
+
+    This extension interacts with:
+
+        EGL_{EXT,KHR}_swap_buffers_with_damage
+        EGL_KHR_partial_update
+        EGL_EXT_output_base
+
+Overview
+
+    This extension extends EGL by allowing late swaps to occur without
+    synchronization to the video frame. This per-surface attribute reduces the
+    visual stutter on late frames and reduces the stall on subsequent frames.
+
+IP Status
+
+    There are no known IP issues.
+
+Issues
```

Comment:
Don't the issues go at the bottom?

Reply:
It's not consistent, but often yes. The section order here is from the [counterpart GLX spec](https://github.com/KhronosGroup/OpenGL-Registry/blob/master/extensions/EXT/GLX_EXT_swap_control_tear.txt).

I would _love_ to see the specs edited for consistency, and possibly even lifted into markdown or rst, but that's not really the fish I'm trying to fry right now.